packages/community-cli-plugin/src/commands/start/attachKeyHandlers.js

```diff
@@ -41,11 +53,15 @@ export default function attachKeyHandlers({
     env: {FORCE_COLOR: chalk.supportsColor ? 'true' : 'false'},
   };
 
+  const reload = throttle(() => {
+    logger.info('Reloading connected app(s)...');
+    messageSocket.broadcast('reload', null);
+  }, reloadTimeout);
+
```

Comment:
Can we switch this behaviour out for `debounce`? Reasoning:
- I reckon the expected user behaviour, more often than not, is to observe an initially requested reload complete within the ~500ms from keypress — and then retry if they made an extra change. _In most cases, I imagine that throttling and emitting a second reload could be unexpected/would interrupt landing on the reloaded app and would be unwanted behaviour._
    - Additionally, Fast Refresh should push an update to the app immediately after a reload, if code was changed during this small interval.
- With this change, I'd also like to shorten the timeout to a flat 500ms — this guards against accidental rapid user keypresses, rather than aiming to synchronise with the end-to-end app reload time (which is variable). (As you've stated, this is a workaround rather than a fix to the underlying issue.)

> [!Note]
> **Sidenote**: It might also make sense to apply the debounce operation to all keys, **OR** tie each command to the completion of each event via Metro's reporter — but leave this with me, I should be working in this file later this week.

Reply:
At first I thought that throttling might have a better user experience as it performs the user initial operation immediately, but now I can see that it may lead to some unwanted behaviour. Thank you for letting me know! I've pushed the commit that changes the throttle to debounce.

Reply:
Oh wait, I'm being a terrible front end engineer. For some reason I hallucinated that `throttle` would re-fire a second keypress 🙈. So long as subsequent calls are _cancelled_, then leading debounce behaviour (i.e. throttle) is better UX, you're right 👍🏻

Let's change back, and accept my eternal thanks? 🥹